controllers/topics-controllers.js

```diff
@@ -0,0 +1,8 @@
+const { selectTopics } = require("../models/topics-models.js")
+
+module.exports.getTopics = (req, res, next) => {
+    return selectTopics().then((topics) => {
+        res.status(200).send(topics.rows);
```

Comment:
Looks good though you should send the data back on a relevant key 🔑 
(this will mean you need to alter your tests slightly)

Reply:
Controller code updated to send array back on an appropriate key. Line 5 of topics-controllers.js.